common/landed_titles/wc_religious_landed_titles.txt

```diff
@@ -380,6 +380,34 @@ d_cult_of_hakkar = {
 }
 
 ### necromancy_group
+# throssian
+d_throssian = {
+	color = { 0 75 100 }
+	color2={ 255 255 255 }
+
+	capital = 3536 # Crimson Forest
+
+	title = "HIGH_DEATHSHAPER_TITLE"
+	title_female = "HIGH_DEATHSHAPER_FEM_TITLE"
+	short_name = yes
+	
+	allow = {
+		always = no
+	}
+
+	# Always exists
+	landless = yes
+
+	# Controls a religion
+	controls_religion = throssian
+
+	religion = throssian
+
+	# Cannot be held as a secondary title
+	primary = yes
+
+	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
+}
```

Comment:
Actually, since it's not used, I would comment this for now. Otherwise, the players will be told that they can create a religious head.